py/client/pydeephaven/session.py

```diff
@@ -78,6 +80,35 @@ def get_token(self):
         return self._token
 
 
+class SharedTicket:
```

Comment:
I am comfortable with leaving it here because the 'e' type tickets are generated on a session object's internal state. So even if we are going to create a more general Ticket class and its sub-classes in the future, they should still stay here.